ec/ecresource/deploymentresource/read.go

```diff
@@ -92,3 +177,24 @@ func deploymentNotFound(err error) bool {
 	// We also check for the case where a 403 is thrown for ESS.
 	return apierror.IsRuntimeStatusCode(err, 403)
 }
+
+// Setting this variable here so that it is parsed at compile time in case
+// any errors are thrown, they are at compile time not when the user runs it.
```

Comment:
```suggestion
// Setting this variable here so that it is parsed at compile time in case
// any errors are thrown, they are at compile time not when the user runs it.
```

This comment is inaccurate, this variable isn't parsed at compile time since it's not a compile time constant. It's executed when this package is first loaded. Granted, if this is invalid it will break in tests, but the comment should reflect that (setting this here so it breaks unit tests) rather than claim it's something that's checked in the compiler.

Reply:
Thanks for the catch up. This is the leftover of the previous code that was moved as-is (though it looks like I renamed it :) ). I'll fix this.